src/Tests/ExecutionPathTracerTests.cs

```diff
@@ -623,5 +668,33 @@ public void MResetZTest()
             var expected = new ExecutionPath(qubits, operations);
             Assert.AreEqual(expected.ToJson(), path.ToJson());
         }
+
+        [TestMethod]
+        public void ForEachMeasureCirc()
+        {
+            var path = GetExecutionPath("ForEachMeasureCirc");
+            var qubits = new QubitDeclaration[]
+            {
+                    new QubitDeclaration(0, 1),
+                    new QubitDeclaration(1, 1),
+            };
+            var operations = new Operation[]
+            {
+                    new Operation()
+                    {
+                        Gate = "measure",
+                        Controls = new List<Register>() { new QubitRegister(0) },
+                        Targets = new List<Register>() { new ClassicalRegister(0, 0) },
+                    },
+                    new Operation()
+                    {
+                        Gate = "measure",
+                        Controls = new List<Register>() { new QubitRegister(1) },
+                        Targets = new List<Register>() { new ClassicalRegister(0, 0) },
```

Comment:
Is `ClassicalRegister(0,0)` correct? Seems it is measuring to the same classical bit, although probably I just don't know exactly what this metadata represents.

Reply:
`ClassicalRegister` has the constructor `ClassicalRegister(int qId, int cId)`. In this case, this means that the classical register is associated with the qubit with ID `0` and its classical bit ID "under qubit 0" is `0`.